cpp/client/ct.cc

```diff
@@ -815,10 +816,10 @@ void WrapEmbedded() {
                cert_trans::NID_ctEmbeddedSignedCertificateTimestampList));
 
   string serialized_scts;
-  CHECK_EQ(Cert::TRUE,
+  CHECK_EQ(true,
```

Comment:
Yeah, saw that, but this looks like a conversion from having been `ValueOrDie()`? It would be equivalent and more friendly if it was `CHECK_EQ(util::Status::OK, ...)` instead.